x-pack/plugins/infra/public/alerting/inventory/components/expression_chart.tsx

```diff
@@ -53,7 +53,6 @@ export const ExpressionChart: React.FC<Props> = ({
         .valueOf(),
       to: moment().valueOf(),
       forceInterval: true,
-      ignoreLookback: true,
```

Comment:
Small clean up. `forceInterval` takes precedence over `ignoreLookback` in https://github.com/elastic/kibana/blob/main/x-pack/plugins/infra/server/routes/snapshot/lib/create_timerange_with_interval.ts#L37.